DataFormats/HcalDigi/interface/HcalUMNioDigi.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H
+#define DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H 1
+
+#include <vector>
+#include <stdint.h>
+#include <ostream>
+
+/** \class HcalUMNioDigi
+  *  
+  * $Date: $
+  * $Revision: $
+  * \author J. Mans - Minnesota
+  */
+class HcalUMNioDigi {
```

Comment:
What's "UMN"? if it's the institution name, it better be avoided (well, maybe there are good precedent examples in CMSSW already, please point me to it).